AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -60,7 +60,7 @@ - (void)_staticInitialize;
 @implementation ASDisplayNode
 
 // these dynamic properties all defined in ASLayoutOptionsPrivate.m
-@dynamic spacingAfter, spacingBefore, flexGrow, flexShrink, flexBasis, alignSelf, ascender, descender, sizeRange, layoutPosition, layoutOptions;
+@dynamic spacingAfter, spacingBefore, flexGrow, flexShrink, flexBasis, alignSelf, ascender, descender, sizeRange, layoutPosition;
```

Comment:
Fantastic to eliminate the most common type of object creation in the framework!